base_layer/core/src/transactions/transaction.rs

```diff
@@ -785,7 +785,7 @@ impl Transaction {
     }
 
     /// This function adds two transactions together. It does not do cut-through. Calling Tx1 + Tx2 will result in
-    /// vut-through being applied.
+    /// cut-through being applied.
```

Comment:
I think this is updated in the tari_script branch. that cut-hrought is not applied, ever.
#2993

Reply:
cool, will remove to avoid conflicts (which is what I think you're getting at :) )